packages/components/.storybook/preview.tsx

```diff
@@ -10,7 +7,7 @@ import { addParameters } from "@storybook/react";
 import { withA11y } from "@storybook/addon-a11y";
 
 addDecorator(withA11y);
-addDecorator(storyFn => <div dir="ltr">{storyFn()}</div>);
+addDecorator((storyFn) => <div dir="ltr">{storyFn()}</div>);
```

Comment:
does prettier not care one way or the other?

Reply:
my prettier did this on save, but if we hate it can add a rule to `.prettierrc`? it looks like the default changed from avoid -> always on the last update

Reply:
I like it 😄 . But it is a change, and either way is fine.

Reply:
It's not a problem, I'm just surprised it wasn't failing prettier before. Probably because it's in .storybook instead of `src/` I'll look into that.

Reply:
Ah, that would make sense